ecommerce/src/main/kotlin/io/hhplus/ecommerce/product/application/EventBatchProcessor.kt

```
@@ -0,0 +1,199 @@
+                        val quantity = (eventMap["quantity"] as Number).toInt()
+                        currentStats.salesCount += quantity
+                    }
+                    eventJson.contains("ProductWished") -> {
+                        currentStats.wishCount += 1
+                    }
+                    eventJson.contains("ProductUnwished") -> {
+                        currentStats.wishCount -= 1 // 찜 해제
+                    }
+                }
+
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)